CoroutineTemplate/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,5 +1,5 @@
 object Versions {
-    const val BUILD_GRADLE_VERSION = "4.2.1"
+    const val BUILD_GRADLE_VERSION = "7.0.2"
```

Comment:
Should we also upgrade Kotlin? https://github.com/nimblehq/android-templates/issues/163#issuecomment-1098919058

If the task is complicated, we can separate it into a new chore @Tuubz as we need to upgrade some test actions also, such as `runBlockingTest` https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-test/MIGRATION.md

Reply:
Oh, right 👍 

I'll create a new ticket for this now, so we can take care of it later 🚀

Reply:
[Done](https://github.com/nimblehq/android-templates/issues/249) ✅